### ISSUE TITLE
Add support for development versions of metamod/sourcemod

### ISSUE THREAD

**Issue by szechyjs** [GameServerManagers/LinuxGSM#1412]
Its quite common for server updates to break metamod and sourcemod. When this occurs its required to install the latest development version to fix the issue.

I suggest adding the ability to install the latest development release of these plugins for those times when the latest stable will not work.

**Comment by zoxovsky** [comment]
People can install it manually,

**Comment by UltimateByte** [comment]
Are you talking about using `mods-update` command?

**Comment by szechyjs** [comment]
Yes, I'm referring to the `mods-install` and `mods-update` commands.
